### PR TITLE
Fix logging calls with wrong arguments

### PR DESCRIPTION
**Pull request by apc999**

### CODE REVIEW

File: core/server/master/src/main/java/alluxio/master/file/DefaultFileSystemMaster.java [hunk 1]
```diff
@@ -3241,9 +3241,9 @@ public void activeSyncMetadata(AlluxioURI path, Collection<AlluxioURI> changedFi
         executorService.invokeAll(callables);
       }
     } catch (InvalidPathException e) {
-      LOG.warn("InvalidPathException during active sync {}", e);
+      LOG.warn("InvalidPathException during active sync {}", e.getMessage());
```
Comment (calvinjia) [inline core/server/master/src/main/java/alluxio/master/file/DefaultFileSystemMaster.java]: `e.toString` is better than `e.getMessage`

Comment (apc999) [reply core/server/master/src/main/java/alluxio/master/file/DefaultFileSystemMaster.java]: Oh, I was following guide here: https://docs.alluxio.io/os/user/stable/en/contributor/Code-Conventions.html?q=getMessage#warn-log-level. Do we want to update the guide too?

Comment (calvinjia) [reply core/server/master/src/main/java/alluxio/master/file/DefaultFileSystemMaster.java]: Yeah we should. The issue with `.getMessage` is sometimes the message is not populated well, and `.toString` is generally just the exception type + message.

**Comment by apc999** [comment]
@calvinjia  PTAL

**Comment by calvinjia** [comment]
alluxio-bot, merge this please.

**Comment by apc999** [comment]
alluxio-bot, cherry-pick this to branch-2.1 please

**Pull request merged**